packages/lodestar-validator/src/api/impl/rest/beacon/beacon.ts

```diff
@@ -17,13 +17,15 @@ import {Json, toHexString} from "@chainsafe/ssz";
 export class RestBeaconApi implements IBeaconApi {
 
   private readonly client: HttpClient;
+  private readonly clientV2: HttpClient;
 
   private readonly logger: ILogger;
 
   private readonly config: IBeaconConfig;
 
   public constructor(config: IBeaconConfig, restUrl: string, logger: ILogger) {
     this.client = new HttpClient({urlPrefix: urlJoin(restUrl, "lodestar")}, {logger});
+    this.clientV2 = new HttpClient({urlPrefix: urlJoin(restUrl, "/v1/beacon")}, {logger});
```

Comment:
Should it be 
```
clientV1 = ... "/v1"
```
or 
```
clientV2 = ... "/v2"
```
?

Reply:
We will drop first client. In future it will be just one haha